db/migrate/20180613221323_fix_column_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class FixColumnNameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :users, :encrypted_password, :password_digest
+    remove_column :users, :salt, :string
+  end
+end
```

Comment:
I probably would've consolidated all 3 of these migrations into the first one since this is all contained in a branch. you can easily run `rake db:rollback` locally if you mess up (or are just playing with) a migration.